content/essentials/intro-to-reactivity.mdx

```diff
@@ -0,0 +1,171 @@
+This dual capability makes signals both readable and mutable, making them integral to the dynamic nature of reactive systems.
+
+#### Getter
+
+The getter is the mechanism responsible for accessing the current value of the signal. 
+When various components or parts of an application need to "read" or retrieve the data stored in a signal, they use the getter. 
+This provides data that is consistent and reliably accessible throughout the system.
+
+#### Setter
+
+On the other hand, the setter is the tool that permits the changing of the signal's value. 
+Any change to the value within a signal, made possible by the setter, can instigate reactive updates in other connected parts of the system. 
+This ensures that the system remains responsive and up-to-date with the latest data changes.
+
+### Effects
+
+Effects and other listeners (such as memos) can be thought of as automated responders.
+They are closely tied to signals and are set into motion whenever there's a change in the signals they observe. 
+Their primary role is to ensure that any modifications in signals are appropriately reflected throughout the system, maintaining its reactivity.
+
+The nature of effects revolves around two functionalities:
+
+- **Observation**: At the heart of every effect is its ability to continuously monitor or "observe" signals. 
+This keeps effects primed to detect changes in the signals they are tied to.
+- **Activation**: Upon recognizing a change in an observed signal, the effect will promptly execute its predefined routine. 
+This can involve tasks like updating the UI, triggering a process, or initiating external functions. 
+Through this activation, effects ensure that the system's reactivity is both immediate and relevant.
+
+## Building a reactive system
+
+Reactive systems are designed around the principle of data-driven reactivity. This is when signals and their dependents, effects, maintain a relationship. This relationship ensures a seamless propagation of data changes throughout the system, guaranteeing its responsiveness and real-time updates.
+
+### 1. Initialization
+
+The start of a reactive system begins with the setup of signals. During this phase, signals are initialized with their respective starting values. These initialized signals serve as the primary data sources that the rest of the system will rely on.
+
+### 2. Subscription
+
+Once signals are established, the next step involves the effects indicating their dependencies. 
+In this phase, effects "subscribe" to specific signals, marking their interest and establishing a link. 
+This subscription keeps the effects aware of any changes to the signals they are tied to.
+
+### 3. Notification
+
+During the notification phase, any alteration in a signal's value sets off a chain reaction. 
+The altered signal proactively notifies all its subscribed effects about the modification. 
```

Comment:
"altered" sounds a bit strange, maybe "updated" or "changed" is more natural, people don't usually say that data structures or variables are "altered".